tools/gomod-required-updater/README.md

```diff
@@ -0,0 +1,31 @@
+# gomod-required-updater
+
+## Installation
+
+The installed binary will be placed in your `$GOPATH/bin` directory. Make sure this directory is in your system's PATH to run the command from anywhere.
+
+```shell
+go install github.com/smartcontractkit/chainlink/tools/gomod-required-updater
+```
+
+## Config
+
+```toml
+# Add any modules in here that you want to update the required version for. Can optionally be provided via the command line flag `-module`.
+modules = [
+    "github.com/smartcontractkit/chainlink/v2"
```

Comment:
Can we derive this from the go.mod? I think we always want to update all locally replaced modules.

Reply:
@jmank88 we could provide a boolean flag like `-self-replace` and it would search go.mod for any modules with a replace that contained the current repo and then update the require line. Thoughts?

Reply:
I don't follow what you mean. I think the effect that we want is for each `go.mod` file, all of the modules listed that are locally replaced should be updated to `<commit hash>`. Here is how `gomods` finds them: https://github.com/jmank88/gomods/blob/main/module.go#L58-L59

Reply:
Got it. I added a new flag: `-update-org-modules`.

Reply:
Made this the default.